### PR TITLE
Fix gke ci upgrade issues

### PR DESCRIPTION
**Pull request by andreas-kupries**
Debug and fix GKE CI upgrade problems.

### CODE REVIEW

File: internal/services/instances.go [hunk 1]
```diff
@@ -374,7 +374,13 @@ func (s *ServiceClient) DeleteForHelmController(ctx context.Context, namespace,
 		metav1.DeleteOptions{},
 	)
 
-	return errors.Wrap(err, "error deleting helm charts")
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			return nil
+		}
+	}
```
Comment (enrichman) [inline internal/services/instances.go:381]: From the `IsNotFound` doc

```
// It supports wrapped errors and returns false when the error is nil.
```

so I guess that we can drop the `err != nil` check.

```suggestion
	if apierrors.IsNotFound(err) {
		return nil
	}
```

File: internal/services/instances.go [hunk 2]
```diff
@@ -389,7 +395,13 @@ func (s *ServiceClient) DeleteAllForHelmController(ctx context.Context, targetNa
 		},
 	)
 
-	return errors.Wrap(err, "error deleting helm charts")
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			return nil
+		}
+	}
```
Comment (enrichman) [inline internal/services/instances.go:402]: Same
```suggestion
	if apierrors.IsNotFound(err) {
		return nil
	}
```

**Review by enrichman — COMMENTED** [review]
Just a small thing. I didn't know that, maybe we can drop some other if/else in the codebase.

File: internal/services/instances.go (hunk 2 above)
Comment (andreas-kupries) [reply internal/services/instances.go]: Commit 713797e45437ab627d57046caa6b4d4ab9640365

**Comment by andreas-kupries** [comment]
FOI - The failed tests pass locally. Together with the flakes I suspect CI load and strangeness.

**Review by enrichman — APPROVED** [review]

**Pull request merged**